testnet-contracts/truffle.js

```diff
@@ -1,3 +1,7 @@
+require("dotenv").config();
```

Comment:
filename should be truffle-config.js, not truffle.js to work with new version of truffle

Reply:
https://github.com/swishlabsco/peggy/pull/2/commits/b4be8d80c6b526799d0fcfc46e5e07389cc40451